src/service/action-impl.js

```diff
@@ -269,21 +272,18 @@ export class ActionService {
         target.tagName.toLowerCase() in ELEMENTS_ACTIONS_MAP_,
         'AMP element or a whitelisted target element is expected: %s', debugid);
 
-    /** @const {!Array<!ActionInvocation>} */
-    const currentQueue = target[ACTION_QUEUE_];
-    if (currentQueue) {
-      dev().assert(
-        isArray(currentQueue),
-        'Expected queue to be an array: %s',
-        debugid
-      );
+    if (target[ACTION_HANDLER_]) {
+      // Already installed
```

Comment:
Let's add `dev().assert()` here, since calling this method twice is really a dev error.